src/main/scala/com/gilt/aws/lambda/AwsLambdaPlugin.scala

```diff
@@ -114,7 +125,11 @@ object AwsLambdaPlugin extends AutoPlugin {
     vpcConfigSecurityGroupIds := None,
     environment := Nil,
     lambdaRuntime := None,
-    packageLambda := sbtassembly.AssemblyKeys.assembly.value
+    packageLambda := sbtassembly.AssemblyKeys.assembly.value,
+    supportedLambdaRuntimes := Set(
```

Comment:
Would suggest making this a Seq/List and instead of a defaultRuntime key, make the default the first element in the list.

Reply:
fixed in f8b1d2e / 1c0d683